qa/tasks/cephfs/test_subvolume.py

```diff
@@ -0,0 +1,207 @@
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from teuthology.exceptions import CommandFailedError
+
+log = logging.getLogger(__name__)
+
+
+class TestSubvolume(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def setUp(self):
+        super().setUp()
+
+    def tearDown(self):
```

Comment:
you can call self.cleanup_test() *before* super.tearDown()

Reply:
noe moved self.cleanup_test() before super.tearDown()